documentation/secure_storage_rpmb.md

```diff
@@ -134,7 +134,7 @@ The FAT is not encrypted.
 
 ## References
 
-- <a name="JEDECeMMC"></a>[1] _Embedded Multi-Media Card (e•MMC) Electrical Standard (5.1)_, JEDEC JESD84-B51, February 2015
+- <a name="JEDECeMMC"></a>[1] _Embedded Multi-Media Card (e-MMC) Electrical Standard (5.1)_, JEDEC JESD84-B51, February 2015
```

Comment:
ok, understood. sorry i highjacked your r-b, indeed you obviously did not review it before.
i'll discard the last commit and handle it seperatly.

I though the markdown format was handy since allowing formatted doc content based on basic text which I tend to relate to ascii. You're right, it's false. Nowadays there is no reason why  plain ascii text is easier to maintain than an utf format text. i'm a bit old school, i need to upgrade my habits.